src/controllerbluetooth.sh

```diff
@@ -0,0 +1,10 @@
+#!/bin/bash
+
+option=$1
+
+if [option = connect]
+    echo 'connect 5C:BA:37:26:6D:9A\nquit' | sudo bluetoothctl
+then
```

Comment:
Why are you doing if then instead of if else? This is always executing both connect and disconnect.